en/frames_multicopter/omnicopter.md

```diff
@@ -0,0 +1,140 @@
+  To avoid conflicts we removed the power pin (right-most pin) from one of the connectors.
+- Telemetry to UART1
+- GPS to UART4
+- RC to UART6
+
+![FC closeup](../../assets/airframes/multicopter/omnicopter/fc_closeup.jpg)
+
+Remarks:
+
+- Make sure the magnetometer is placed away from power lines.
+  We ended up placing it to the bottom of the center-piece with a 4cm padding styrofoam.
+- Put some tape on the barometer (without taping the opening!) to avoid any influence from light.
+- We advise that you glue the frame after initial test flights (we did not, and it may work without doing so).
+
+
+## Software Configuration
+
+### ESC
+
+First, configure the ESCs to 3D mode (bidirectional).
+
+We had issues with the stock ESC settings in 3D mode and had to change the ESC settings (when trying to switch direction, a motor sometimes did not start anymore until the ESC was rebooted).
+
+You can change the ESC settings using Betaflight on the flight controller and then use pass-through mode and the BL Heli suite (make sure there is an airframe with 8 motors configured in Betaflight).
+
+These are the settings:
+![ESC settings](../../assets/airframes/multicopter/omnicopter/esc_settings.png)
+
+In particular:
+- set the Motor Direction to **Bidirectional Soft**
+- increase the Rampup Power to **100%** (this is conservative and might reduce efficiency)
+
+:::note
+Make sure the motors do not overheat with the changed settings.
+:::
+
+### PX4 Configuration
+
+- In the[Airframe](../config/airframe.md) configuration select a _generic multicopter_. 
+- Enable [SYS_CTRL_ALLOC](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC)
+- Use an [arming switch](../advanced_config/prearm_arm_disarm#arming-button-switch).
+  Do not use stick arming.
+- In the [Actuator](../config/actuators.md) configuration:
+  - Select DShot as the as output protocol on all eight outputs
+  - Configure the motor geometry according to this:
+
+    ![Motor Config](../../assets/airframes/multicopter/omnicopter/motors_configuration.png)
+  
+    We used the following convention: the motors are facing the direction of where the axis points to.
+    The spinning direction matches the direction of positive thrust (moving the motor slider up).
+    Make sure to use the right props, as there is a CCW and a CW version.
+  
+- Parameters:
+  - Change the desaturation logic for better attitude tracking: set [CA_METHOD](../advanced_config/parameter_reference.md#CA_METHOD) to 0.
+  - Disable failure detection: set [FD_FAIL_P](../advanced_config/parameter_reference.md#FD_FAIL_P) and [FD_FAIL_R](../advanced_config/parameter_reference.md#FD_FAIL_R) to 0.
+  
+TODO: params file + tuning params
+
+<!-- TODO: video -->
+
+## Simulation
+
+There is an omnicopter gazebo simulation target:
+
+```sh
+make px4_sitl gazebo_omnicopter
+```
+
+![Gazebo Sim](../../assets/airframes/multicopter/omnicopter/gazebo.png)
+
+## Remarks
+
+Some general remarks:
+- The hover throttle is about 30%.
```

Comment:
How this set?

Reply:
It does not need to be

Reply:
OK. Let me know when you guys are done with this doc.